api/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/hashicorp/consul/api
 
 go 1.19
 
-//replace github.com/hashicorp/consul/sdk => ../sdk
+replace github.com/hashicorp/consul/sdk => ../sdk
```

Comment:
For consistency, I also updated this one even though I didn't change `sdk`; see comment [here](https://github.com/hashicorp/consul/pull/17432/files#r1202563919)